gen/definitions/profile_parcels/service_object_tracker.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: Service Object Tracker
+rest_endpoint: /v1/feature-profile/sdwan/service/%v/objecttracker
+minimum_version: 20.12.0
+test_tags: [SDWAN_2012]
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_service_feature_profile.test.id
+  - model_name: objectId
+    tf_name: object_tracker_id
+    mandatory: true
+    example: 10
+    test_value: '10'
+  - model_name: objectTrackerType
+    tf_name: tracker_type
```

Comment:
The previous parameter name is "objectId"/"object_tracker_id" and this one is "objectTrackerType"/"tracker_type". I know our API schema is also not aligned on this, but would it make sense to use either "objectTrackerType"/"objectTrackerId" or "TrackerType/TrackerId" to have it more consistent? And then use same rule for tf_name?

Reply:
I do agree but that is not something we can change without the API being updated. We cannot change the model_name unfortunately as this is based on what the API expects. The tf_name's were based on the names displayed in the UI but let me know which you think is more clear and I can change it
![object-tracker-ui](https://github.com/user-attachments/assets/2130c4c9-ad75-4800-a7cb-8357adc68232)

Reply:
If you are OK with that, let's do "object_tracker_id" and "object_tracker_type" to keep it more consistent.